HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -287,11 +287,38 @@ public IActionResult ManualAddress_Post(ManualAddressViewModel viewModel)
     [HttpGet("income")]
     public IActionResult HouseholdIncome_Get()
     {
-        return RedirectToAction(nameof(StaticPagesController.Index), "StaticPages");
+        var questionnaire = questionnaireService.GetQuestionnaire();
+        var viewModel = new HouseholdIncomeViewModel()
+        {

```

Comment:
We need to set the IncomeBand from the questionnaire here so that when people go back, or change their answer it shows the current answer.